profile/src/main/java/org/openedx/profile/presentation/settings/SettingsScreenUI.kt

```diff
@@ -75,6 +84,8 @@ import org.openedx.profile.R as profileR
 internal fun SettingsScreen(
     windowSize: WindowSize,
     uiState: SettingsUIState,
+    iapUiState: IAPUIState?,
+    fragmentManager: FragmentManager,
```

Comment:
please remove as described upper, replace with an new action in SettingsScreenAction

Reply:
addressed.